### DISCUSSION TITLE
Define random generated password max length through installer flag

### DISCUSSION THREAD

**Discussion by pacman366** [rustdesk/rustdesk#1126] [General]
Hi team, loving the work you're all doing here. I think it would be awesome to see an installer flag similar to the "--silent-install" one that would define the max lengths of passwords rustdesk generates. For example, Something like  "--password-length=10" that would cause a fresh install to generate 10 character long passwords. I'm a bit concerned 6 characters consisting of just letters may not be enough entropy to prevent brute force attacks, Especially now with the new local network discovery feature in mid to large scale deployments. Granted, I dont thing this is something that's a very likely vector of attack for Rustdesk currently, but if the software becomes more popular over time I can see it becoming an issue.

**Comment by rustdesk** [comment]
PR welcome.
